README.md

```diff
@@ -14,6 +14,7 @@ The goal of this image is to easily run [Minecraft Overviewer](https://overviewe
 docker run \
   --rm \
   -e MINECRAFT_VERSION=1.11.2 \
+  -e API_KEY=<GMaps API Key>
```

Comment:
Minor preference: can you change `GMaps` to `Google Maps` as that's more clear for users.